test/unit/compileProtos.ts

```diff
@@ -61,18 +60,26 @@ describe('compileProtos tool', () => {
     process.chdir(cwd);
   });
 
-  it('compiles protos to JSON', async () => {
+  it('compiles protos to JSON, JS, TS', async () => {
     await compileProtos.main([
       path.join(__dirname, '..', '..', 'test', 'fixtures', 'protoLists'),
     ]);
-    const expectedResultFile = path.join(resultDir, 'protos.json');
-    assert(fs.existsSync(expectedResultFile));
+    const expectedJsonResultFile = path.join(resultDir, 'protos.json');
+    const expectedJSResultFile = path.join(resultDir, 'protos.js');
+    const expectedTSResultFile = path.join(resultDir, 'protos.d.ts');
+    assert(fs.existsSync(expectedJsonResultFile));
+    assert(fs.existsSync(expectedJSResultFile));
+    assert(fs.existsSync(expectedTSResultFile));
 
-    const json = await readFile(expectedResultFile);
+    const json = await readFile(expectedJsonResultFile);
     const root = protobuf.Root.fromJSON(JSON.parse(json.toString()));
-
     assert(root.lookup('TestMessage'));
     assert(root.lookup('LibraryService'));
+
+    const js = await readFile(expectedJSResultFile);
+    const ts = await readFile(expectedTSResultFile);
+    assert(js.toString());
```

Comment:
Can we check that both `js` and `ts` contain `TestMessage` and `LibraryService`? For example, if parameters to `pbts` are incorrect, it will still generate a `.d.ts` file but it will only contain one import line and no types.